functions/snippets/helloworld/src/main/scala/com/example/functions/helloworld/ScalaHelloBackground.scala

```diff
@@ -1,9 +1,10 @@
-package com.example.functions
+package com.example.functions.helloworld
 
 // [START functions_helloworld_background]
-import com.google.cloud.functions.{BackgroundFunction, Context, HttpRequest}
 import java.util.logging.Logger
 
+import com.google.cloud.functions.{BackgroundFunction, Context, HttpRequest}
+
 class ScalaHelloBackground extends BackgroundFunction[HttpRequest] {
```

Comment:
We can address it later, but this seems wrong to me. `HttpRequest` isn't the sort of class you would expect to see as the payload of an event.